lib/features/helpers/hash_functions.dart

```diff
@@ -0,0 +1,16 @@
+import 'dart:convert';
+import 'dart:async';
+import 'package:crypto/crypto.dart';
+import 'package:flutter_dotenv/flutter_dotenv.dart';
+
+Future<String> hashString(String input) async{
```

Comment:
```suggestion
import 'package:crypto/crypto.dart';
import 'package:flutter_dotenv/flutter_dotenv.dart';

Future<String> hashString(String input) {
```
Функции нет смысла быть асинхронной. Убери определение из функции (6-ая строчка) и импорт библиотеки (2-ая строчка) и, считай, ПР готов :)